hack/get_ci_vm.sh

```diff
@@ -0,0 +1,215 @@
+        print "{0}=\"{1}\"".format(k, v),
+    '
+}
+
+parse_args(){
+    echo -e "$USAGE_WARNING"
+
+    if [[ "$USER" =~ "root" ]]
+    then
+        show_usage "This script must be run as a regular user."
+    fi
+
+    ENVS="$(get_env_vars)"
+    IMAGE_NAME="$1"
+    if [[ -z "$IMAGE_NAME" ]]
+    then
+        show_usage "No image-name specified."
+    fi
+
+    ENVS="$ENVS SPECIALMODE=\"$SPECIALMODE\""
+    SETUP_CMD="env $ENVS $GOSRC/contrib/cirrus/setup.sh"
+    VMNAME="${VMNAME:-${USER}-${IMAGE_NAME}}"
+    CREATE_CMD="$PGCLOUD compute instances create --zone=$ZONE --image-project=libpod-218412 --image=${IMAGE_NAME} --custom-cpu=$CPUS --custom-memory=$MEMORY --boot-disk-size=$DISK --labels=in-use-by=$USER $VMNAME"
+    SSH_CMD="$PGCLOUD compute ssh $SSHUSER@$VMNAME"
+    CLEANUP_CMD="$PGCLOUD compute instances delete --zone $ZONE --delete-disks=all $VMNAME"
+}
+
+##### main
+
+[[ "${STORAGEROOT%%${STORAGEROOT##$HOME}}" == "$HOME" ]] || \
+    show_usage "Repo clone must be sub-dir of $HOME"
+
+cd "$STORAGEROOT"
+
+parse_args "$@"
+
+# Ensure mount-points and data directories exist on host as $USER.  Also prevents
+# permission-denied errors during cleanup() b/c `sudo podman` created mount-points
+# owned by root.
+mkdir -p $TMPDIR/${STORAGEROOT##$HOME}
+mkdir -p $TMPDIR/.ssh
+mkdir -p {$HOME,$TMPDIR}/.config/gcloud/ssh
+chmod 700 {$HOME,$TMPDIR}/.config/gcloud/ssh $TMPDIR/.ssh
+
+cd $STORAGEROOT
+
+# Attempt to determine if named 'storage' gcloud configuration exists
+showrun $PGCLOUD info > $TMPDIR/gcloud-info
+if egrep -q "Account:.*None" $TMPDIR/gcloud-info
+then
+    echo -e "\n${YEL}WARNING: Can't find gcloud configuration for 'storage', running init.${NOR}"
+    echo -e "         ${RED}Please choose '#1: Re-initialize' and 'login' if asked.${NOR}"
+    echo -e "         ${RED}Please set Compute Region and Zone (if asked) to 'us-central1-b'.${NOR}"
+    echo -e "         ${RED}DO NOT set any password for the generated ssh key.${NOR}"
+    showrun $PGCLOUD init --project=$PROJECT --console-only --skip-diagnostics
+
+    # Verify it worked (account name == someone@example.com)
+    $PGCLOUD info > $TMPDIR/gcloud-info-after-init
+    if egrep -q "Account:.*None" $TMPDIR/gcloud-info-after-init
+    then
+        echo -e "${RED}ERROR: Could not initialize 'storage' configuration in gcloud.${NOR}"
+        exit 5
+    fi
+
+    # If this is the only config, make it the default to avoid persistent warnings from gcloud
+    [[ -r "$HOME/.config/gcloud/configurations/config_default" ]] || \
+        ln "$HOME/.config/gcloud/configurations/config_storage" \
+           "$HOME/.config/gcloud/configurations/config_default"
+fi
+
+# Couldn't make rsync work with gcloud's ssh wrapper: ssh-keys generated on the fly
+TARBALL=$VMNAME.tar.bz2
+echo -e "\n${YEL}Packing up local repository into a tarball.${NOR}"
+showrun --background tar cjf $TMPDIR/$TARBALL --warning=no-file-changed --exclude=.git --exclude-vcs-ignores -C $STORAGEROOT .
```

Comment:
This is wrong, do not --exclude=.git
We need git when we login the --exclude-vcs-ignores is enough.

Reply:
We do this in libpod because it saves time in the transfer, but if it's inconvenient here I can strip it, np.

Reply:
kk, got it, and `--force` pushed.